### PR TITLE
Cool new commands

### PR DESCRIPTION
**Pull request by C10udburst**
- .inventory
- .damage
- .plugins
- .server

### CODE REVIEW

File: src/main/java/minegame159/meteorclient/commands/commands/Dmg.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                ChatUtils.error("You are in creative");
+                return SINGLE_SUCCESS;
+            }
+            DamagePlayer(amount);
+            return SINGLE_SUCCESS;
+        }));
+
+    }
+    
+    private void DamagePlayer(int amount) {
```
Comment (seasnail8169) [inline src/main/java/minegame159/meteorclient/commands/commands/Dmg.java:37]: following standards, method names should start lowercase

File: src/main/java/minegame159/meteorclient/commands/commands/Plugins.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                        if (!plugins.contains(pluginName)) {
+                            plugins.add(pluginName);
+                        }
+                    }
+                }
+                ;
+                Collections.sort(plugins);
+
+                if (!plugins.isEmpty()) {
+                    String s = "Plugins \u00A77(\u00A78" + plugins.size() + "\u00A77): \u00A7c"
```
Comment (seasnail8169) [inline src/main/java/minegame159/meteorclient/commands/commands/Plugins.java:82]: Here, and otherplaces, meteor has methods in place for string formatting, please use them. If you are unsure check ChatUtils.

**Comment by MineGame159** [comment]
Definitively rename .dmg to .damage and maybe .invsee to .inventory or something more easier to understand.

**Comment by seasnail8169** [comment]
> Definitively rename .dmg to .damage and maybe .invsee to .inventory or something more easier to understand.

definitely != definitively 

come on @MineGame159 i taught you this 😔

**Comment by MineGame159** [comment]
> > Definitively rename .dmg to .damage and maybe .invsee to .inventory or something more easier to understand.
> 
> definitely != definitively
> 
> come on @MineGame159 i taught you this 😔

:weary:

**Pull request merged**